src/themes/masschallenge/elements/segment.overrides

```diff
@@ -4,4 +4,8 @@
 
 .ui.basic.unpadded.segment {
 	padding: 0px;
+}
+
+.ui.lightblue.segment {
+  background-color: #F1F3F5 !important;
```

Comment:
I don't like using `!important` here. It is possible we can do without this change, please remove.